pandas/tests/io/test_html.py

```diff
@@ -1113,9 +1113,8 @@ def test_works_on_valid_markup(self, datapath):
     @pytest.mark.slow
     def test_fallback_success(self, datapath):
         banklist_data = datapath("io", "data", "html", "banklist.html")
-        self.read_html(
-            banklist_data, match=".*Water.*", flavor=["lxml", "html5lib"]
-        )  # pylint: disable=redundant-keyword-arg
+        # pylint: disable-next=redundant-keyword-arg
+        self.read_html(banklist_data, match=".*Water.*", flavor=["lxml", "html5lib"])
```

Comment:
this one's also in the `intentionally turned off section`, I think we can just remove the comment

do you want to run try running https://github.com/MarcoGorelli/pylint-reenable to see if there's other redundant `pylint: disable` comments we can remove?

Reply:
not yet - you could do
```
git ls-files | grep '\.py$' | xargs pylint-reenable
```